python/ray/data/_internal/plan.py

```diff
@@ -635,12 +636,18 @@ def execute(
                         stats_summary_string,
                     )
 
+            # Retrieve memory-related stats from ray.
+            reply = get_memory_info_reply(get_state_from_address(None))
+            if reply.store_stats.spill_time_total_s > 0:
+                stats.bytes_spilled = int(reply.store_stats.spilled_bytes_total)
```

Comment:
nit: could we name it something like `global_bytes_spilled`, to indicate that this is global (and not just ray data)?